auth-web/tests/unit/test-utils/test-data/affiliations.ts

```diff
@@ -122,6 +122,16 @@ export const businesses: Business[] = [
     corpType: { code: CorpTypes.BENEFIT_COMPANY },
     status: 'HISTORICAL'
   },
+  // Amalgamation Application
+  {
+    businessIdentifier: 'TC6weERhbH',
+    corpType: {
+      code: CorpTypes.AMALGAMATION
+    },
+    corpSubType: {
+      code: CorpTypes.AMALGAMATION
```

Comment:
I'm not sure how this list is used...

Should this subtype be REGULAR?

Reply:
No Sev, but you're right. It shouldn't be the same. The code is correct but the subType should be the CompanyType. I'll be fixing that. Good catch!

Reply:
nvm

Reply:
Sev, reverting the change. SonarCloud is going crazy about mixing enum values with strings.